tox.ini

```diff
@@ -10,7 +10,12 @@ commands =
     coverage combine
     coverage report
     coverage xml
-passenv = DATABASE_URL PORT aws_access_key_id aws_secret_access_key
+passenv = 
+    DATABASE_URL PORT 
+    aws_access_key_id 
+    aws_secret_access_key 
+    is_aws_mturk_requestor 
+    mturk_worker_id
```

Comment:
I went with a module `tests/config.py` which will be excluded from version control to store the worker ID. This means it can be retrieved from either the env variable (which is what Travis CI will do), or from this file. An empty template file *is* in version control, so the template can just be copied to create the correct file. This is detailed in the docs on running tests.

Tests now assume that the AWS account credentials acquired via the Configurator are for a valid MTurk Requester.